.travis.yml

```diff
@@ -0,0 +1,4 @@
+sudo: false
+language: node_js
+node_js:
+  - "8"
```

Comment:
Since we are still supporting Node.js 6.x in loopback-next, I think we should run CI on 6.x for this repository too.